modules/gitrepo/walk_gogit.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build gogit
+
+package gitrepo
+
+import (
+	"context"
+
+	repo_model "code.gitea.io/gitea/models/repo"
+
+	"github.com/go-git/go-git/v5/plumbing"
+)
+
+// WalkReferences walks all the references from the repository
+// refType should be empty, ObjectTag or ObjectBranch. All other values are equivalent to empty.
```

Comment:
Which reftype?
Do you mean
```suggestion
// refname is empty, ObjectTag or ObjectBranch. All other values should be treated as equivalent to empty.
```